dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -55,12 +56,19 @@
       {% do exceptions.raise_compiler_error(empty_unique_key) %}
     {% endif %}
 
+    {% set inc_predicates_not_list -%}
```

Comment:
You can do the set inside the if if we want to not set in any case

Reply:
I've made this change. Let me know what you think.